### PR TITLE
Testing slack integration

### PR DESCRIPTION
**Pull request by stilesj-uchicago**
### Improvements
Testing a second PR

### CODE REVIEW

File: README.slack.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Github slack integration: https://github.com/integrations/slack
+1. Add the App to Slack
```
Comment (mpsolano) [inline README.slack.md:2]: Code level comment

**Review by mpsolano — COMMENTED** [review]
Comment without approval

File: README.slack.md (hunk 1 above)
Comment (stilesj-uchicago) [reply README.slack.md:2]: @mpsolano  Do you get called out in Slack?

Comment (stilesj-uchicago) [reply README.slack.md:2]: Or does it have to be @mikesolano ?

File: README.slack.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+1. Login `/github signin`
+1. Register a repository
+	```
+	/github subscribe NCI-GDC/indexclient pulls reviews comments
+	/github unsubscribe NCI-GDC/indexclient issues releases deployments
```
Comment (stilesj-uchicago) [inline README.slack.md:7]: This is a review comment.

**Comment by stilesj-uchicago** [comment]
Top level comment

**Review by mpsolano — COMMENTED** [review]
Adding top level comment on this PR

**Comment by stilesj-uchicago** [comment]
Are my top level comments being ignored?

File: README.slack.md (hunk 1 above)
Comment (stilesj-uchicago) [inline README.slack.md:2]: Can I start my own review?

File: README.slack.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Github slack integration: https://github.com/integrations/slack
+1. Add the App to Slack
+1. Login `/github signin`
+1. Register a repository
+	```
+	/github subscribe NCI-GDC/indexclient pulls reviews comments
```
Comment (stilesj-uchicago) [inline README.slack.md:6]: More review comments.

**Review by stilesj-uchicago — COMMENTED** [review]
Jason completed a review!

**Pull request closed without merging**